internal/summary/summary.go

```diff
@@ -12,8 +13,16 @@ type Summary struct {
 }
 
 type Service struct {
-	Name string
-	Path string
+	Name    string
+	Path    string
+	k8sData []k8sDatum
+}
+
+type k8sDatum struct {
```

Comment:
ha this is pedantic but also grammatically correct... except that imo this struct holds multiple pieces of data? 🤷‍♀️